scripts/commands/view-list.php

```diff
@@ -14,7 +15,7 @@
 $query = Container::getInstance()->getShoppingListDetailQuery();
 
 try {
-    $list = $query->execute($slug);
+    $list = $query->execute(new GetShoppingListDetailRequest('supplies', 'all'));
```

Comment:
Required: if you try executing this script with a shopping list slug that is not `supplies` this will not work. The slug comes from user input, and shouldn't be hard-coded.